CONTRIBUTING.md

```diff
@@ -90,6 +90,18 @@ See
 [`tox.ini`](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/tox.ini)
 for more detail on available tox commands.
 
+### Run tests with tox-uv (experimental)
+
+You can speedup the tests by run tox with `uv` support using the following commands:
+```sh
+$ pip install tox-uv
+```
+
+```sh
+# Will use uv
+$ TOX_ENV="uv" tox -e py312-test-instrumentation-aiopg
```

Comment:
Not a big deal, but I'm wondering what the motivation was behind the naming of `TOX_ENV`? I might have expected this to be named `PIP_RUNNER` or `PIP_REPLACEMENT` or something.

Reply:
Before, I was separating UV things in a dedicated test and forgot to change it. Will change to another name. `PIP_RUNNER` or `TOX_RUNNER` sounds good to me. Tox defines two runners: virtualenv, and uv-venv-runner